test/include/seqan3/test/testing_list.hpp

```diff
@@ -0,0 +1,43 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2018, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2018, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#pragma once
+
+#include <gtest/gtest.h>
+
+#include <seqan3/core/metafunction/template_inspection.hpp>
+
+namespace seqan3
+{
+
+//!\cond DEV
+/*!\brief Transforms any type list into a google type list, i.e. ::%testing::Types.
+ * \ingroup core
+ * \tparam  type_list  The type list.
+ *
+ * \details
+ *
+ * You should use this if you would have to redefine every type in a second type list just to test the same types in
+ * gtest's `TYPED_TEST` and in one of your test cases. This reduces redundancy and errors when keeping the types
+ * up-to-date and in sync.
+ *
+ * \attention
+ * Google type lists can't easily be transformed by seqan3::detail::transfer_template_args_onto to a seqan3::type_list,
```

Comment:
Why would anyone want this?

Reply:
To reuse the types.

Reply:
But why would you transfer the types from the google interface to your own type? 
I think you can always go from your type to google type list and there is no need to convert the other way around. Or am I misreading the sentence?